programs/server/config.xml

```diff
@@ -1688,17 +1688,14 @@
     -->
 
     <send_crash_reports>
-        <!-- Changing <enabled> to true allows sending crash reports to -->
+        <!-- Setting <enabled> to true allows sending crash reports to -->
         <!-- the ClickHouse core developers team via Sentry https://sentry.io -->
         <!-- Doing so at least in pre-production environments is highly appreciated -->
-        <enabled>false</enabled>
-        <!-- Change <anonymize> to true if you don't feel comfortable attaching the server hostname to the crash report -->
-        <anonymize>false</anonymize>
-        <!-- Default endpoint should be changed to different Sentry DSN only if you have -->
-        <!-- some in-house engineers or hired consultants who're going to debug ClickHouse issues for you -->
+        <enabled>true</enabled>
```

Comment:
Should we also change the defaults here? https://github.com/ClickHouse/ClickHouse/blob/1f93c8a2b108c968437865b43a12226818ecc9eb/src/Daemon/SentryWriter.cpp#L86